FrontEnd/src/components/basics/Modal/Modal.tsx

```
@@ -30,7 +42,28 @@ const Modal: FunctionComponent<ModalProps> = ({
       disableBackdropClick
     >
       <div className={cn(styles.modal, styles[size], className)}>
-        <>{children}</>
+        <div>
+          <Button variant="wrapper" className={styles.close} onClick={onClose}>
+            <miscIcons.orangeX />
+          </Button>
+        </div>
+
+        {(src || alt) && (
```

this should be only `src` not `src || alt`. consider if someone passes through an `alt` but not a `src`. then the image would be rendered without a `src` (not good).

if we think about it, we actually want to throw an error if an `alt` is passed through without a `src`. We can do this with discriminated types (difficult to do) OR better, we can do it by combining the `src` and `alt` into one object. Consider changing the src/alt props into an "image" prop which is an object { src: string; alt?: string }. Then you can handle the image prop by making it optional. This way, someone can pass nothing for the image, a src for the image, or both a src and alt.

Even better (this is optional, it's a bit harder to do), you can make the image prop like this:
`image?: string | ImageTypeName; // where ImageTypeName is the { src: string, alt?: string }`
so the user can pass only the src through the image (`image={srcVariable}`) or with an alt (`image={{ src: "", alt: "" }}`)

---

Also, what's the image supposed to be for? maybe make the "image" prop a more specific name to describe what it is better. Also, if the prop isn't clear, you should add an end of line comment describing what it's for (or a component header, which I've added for some but not all the components I wrote -- i just forgot to lol)